strongbox-server/src/main/java/core/KeyStoreManager.java

```diff
@@ -13,35 +13,72 @@
 import java.util.Enumeration;
 
 /**
+ * Add methods to manage a KeyStore
```

Comment:
Plutôt "This class contains methods to manage a KeyStore"
Tu peux préciser qu'on peut rechercher, ajouter et supprimer des clés (mais ça j'ai pas encore écris le code)